### PR TITLE
setcookie and getcookie wrapper function for JS #11688 

### PR DESCRIPTION
**Pull request by zaverichintan**
Wrapper function for setting and getting cookies in JS.
Signed-Off-By: chintan zaveri zaveri_chintan@rediff.com

### CODE REVIEW

File: libraries/Config.php [hunk 1]
```diff
@@ -18,6 +18,7 @@
  * Indication for error handler (see end of this file).
  */
 $GLOBALS['pma_config_loading'] = false;
+//chintanzaveri issue11688
```
Comment (nijel) [inline libraries/Config.php]: I guess this should not be included...

File: js/functions.js [hunk 2]
```diff
@@ -4967,3 +4972,39 @@ AJAX.registerOnload('functions.js', function(){
...
+    var jsonDecoded = JSON.parse(arr);
+    var value = jsonDecoded[String(key)];
+}
+
+if (typeof value =='undefined'){
+    return(null);
+}
+else
+    return(value);
+}
```
Comment (nijel) [inline js/functions.js]: Can you please properly indent the code?

**Comment by zaverichintan** [comment]
@nijel kindly over view this, This change does not produce any errors and is tested.

**Comment by nijel** [comment]
This really makes no sense without the PHP side of the patch, which is not yet in state it could be merged (or work).

**Comment by zaverichintan** [comment]
IMHO both the parts can be done separatly as in JS part there was no wrapper function to set cookie and get cookie which made it clear to develop. 
PHP part fails due to no usage of existing wrapper function,which can't be done using my commit even. 
So we can just decrease the JS set Cookies.

**Comment by nijel** [comment]
Closing as there is no matching PHP code for this.

**Comment by zaverichintan** [comment]
Wrapper function for cookies not being used everywhere. Cookies are set and accessed directly using php setcookie function, which makes it difficult to combine multiple cookies.

**Pull request closed without merging**